### ISSUE TITLE
Réaliser un contributing.md pour GeoNature

### ISSUE THREAD

**Issue by Adrien-Pajot** [PnX-SI/GeoNature#1406]
Avec le nombre de contributeurs, contributrices et la méthode de travail au sein de cette communauté, avec @corentinlange, on pense que mettre en place un contributing.md devient incontournable. 
Ce contributing permettrait de cadrer la manière d'arriver sur le répo et d'y contribuer pour garder une première cohérence technique. 
Je veux bien me proposer pour le faire mais j'ai d'abord besoin de vos avis là-dessus.

**Comment by camillemonchicourt** [comment]
Oui, c'est important, voir sur le sujet : https://guides.etalab.gouv.fr/logiciels/#ouvrir-comment-encourager-les-contributions
Ou s'appuyer sur d'autres exemples dans d'autres dépôts.
Voir aussi les éléments dans la doc Développeur de GN : http://docs.geonature.fr/development.html#pratiques-et-regles-de-developpement
